inductiva/__init__.py

```diff
@@ -26,3 +26,31 @@
 urllib3_logger.setLevel(logging.CRITICAL)
 
 __version__ = "0.4.0"
+
+
+def _check_for_available_package_update():
+    # pylint: disable=import-outside-toplevel
+    from .localization import translator as t
+    import urllib3
+    import json
+    import sys
+
+    new_version = __version__
+
+    try:
+        http = urllib3.PoolManager()
+        resp = http.request("GET",
+                            "https://pypi.org/pypi/inductiva/json",
+                            headers={"Accept": "application/json"})
+        json_response = json.loads(resp.data)
+        new_version = json_response["info"]["version"]
+
+    except Exception as ex:  # pylint: disable=broad-exception-caught
+        logging.warning(t["failed-update-check"], ex)
+
+    if new_version != __version__:
+        msg = t["upgrade-available"].format(new_version, __version__)
+        sys.stderr.write(msg + "\n")
```

Comment:
This should write to the stdout, no? Notice, that this check doesn't mean things won't work, just that the user won't have the latest features. But scripts they had previously wrote, should still work.

Reply:
yes, you're correct. It is only printing to stdout so that redirections of the stdout to files do not have this message included. Ex:

```bash
$ python -c 'import inductiva; print(inductiva.__path__)' > tmp.txt
Version 0.4.0 of the 'inductiva' package is available (the currently installed version is 0.3.0).
Consider upgrading to the newer version using:
  pip install --upgrade inductiva
$ cat tmp.txt
['build/__editable__.inductiva-0.3.0-py3-none-any/inductiva']
```